src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -387,7 +389,23 @@ public MappingsSaver mappingsSaver() {
 
     @Override
     public Notifier notifier() {
-        return new ConsoleNotifier(verboseLoggingEnabled());
+        String notifier = (String) optionSet.valueOf(NOTIFIER);
+        if ("ConsoleNotifier".equals(notifier)) {
+            return new ConsoleNotifier(verboseLoggingEnabled());
+        }
+        if ("Slf4jNotifier".equals(notifier)) {
+            return new Slf4jNotifier(verboseLoggingEnabled());
+        }
+        else {
+            try {
+                ClassLoader loader = Thread.currentThread().getContextClassLoader();
+                Class<?> cls = loader.loadClass(notifier);
+                return (Notifier)cls.newInstance();
+            } catch (Exception e) {
+                return throwUnchecked(e, null);
```

Comment:
In my opinion, It it better to add additional information in the exception. For instance: "Cannot load notifier by its name: full_name_of_the_notifier_class"